README.md

```diff
@@ -73,15 +73,7 @@ Other operating systems will have a similar set of commands. Please check Docker
 Now we're in an environment that has everything set up, and we start by first initializing the cluster and then firing up the node:
```

Comment:
replace with (your favorite variation of):
`Setting up Cockroach is easy, but starting a test node is even easier: All it takes is running`